src/makeRobot.js

```diff
@@ -38,7 +38,100 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robotObject = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
when variables have the same names as properties, no need to write each name twice, use this shorthand
```suggestion
    name,
    wheels,
    version,
```